### ISSUE TITLE
Cross origin resource sharing issues

### ISSUE THREAD

**Issue by temilaj** [acekyd/gathered#12]
Application refuses to load meetup data after user logs in. Returns the error. 

```
cannot load https://api.meetup.com/members/self?access_token=ce1faf2bcf5bff8bf44903a4b6f68751. No 'Access-Control-Allow-Origin' header is present on the requested resource. Origin 'http://localhost:8100' is therefore not allowed access.
```

**Comment by acekyd** [comment]
@temilaj on mobile or on browser? If it's browser install the CORS plugin at https://chrome.google.com/webstore/detail/allow-control-allow-origi/nlfbmbojpeacfghkpbjhddihlkkiljbi?hl=en

**Comment by temilaj** [comment]
@acekyd Thanks man. 
Do you want to include this in the documentation, so other users/ potential contributors can easily figure out how to setup the app? 
Also I solved this on my machine without the plugin.
Shall I raise a PR?

**Comment by acekyd** [comment]
Oh without the plugin? By all means do raise a PR 😄  @temilaj

**Comment by temilaj** [comment]
Alright, I will

**Comment by acekyd** [comment]
Fixed with #14
